testutils/test_server.go

```diff
@@ -67,12 +71,17 @@ func NewTestServer(t testing.TB, opts *ChannelOpts) *TestServer {
 		ch: comparableState(ch),
 	}
 
-	return &TestServer{
+	ts := &TestServer{
 		TB:            t,
 		channels:      []*tchannel.Channel{ch},
 		channelStates: states,
 		postFns:       opts.postFns,
 	}
+
+	if opts.IncludeRelay {
```

Comment:
Yep, that's what I had in mind. Once relaying is working well, I'd like to make that behavior opt-out instead of opt-in.